backend/src/graphql/model/PendingContribution.ts

```diff
@@ -2,7 +2,7 @@ import { ObjectType, Field, Int } from 'type-graphql'
 import Decimal from 'decimal.js-light'
 
 @ObjectType()
-export class PendingCreation {
+export class PendingContribution {
```

Comment:
does this Object only be used for real pending to say not booked contributions or for generally all contributions independent from confirmed, booked, deleted?

Reply:
These are only the contributions that require confirmation and that are not confirmed yet